UsingSeal.md

```diff
@@ -229,12 +239,47 @@ curl http://0.0.0.0:2024/v1/service
 curl -H 'Content-Type: application/json' -d '{"ptb":"AwAgv7n4Pj/owIGk+nMEXJ4KbXvVytgdurspb4BZJ8RdY6MAISDAbM35ybJGgprxCy0H1MuWKsGvRO5zQfk4jn+oAcufAwEBwGzN+cmyRoKa8QstB9TLlirBr0Tuc0H5OI5/qAHLnwOtCKsRAAAAAAABAG0lMvMLkGCIOUyRinodOfXWx1Bxm0JCFS/6RhBLoEUxCXdoaXRlbGlzdAtrbXNfYXBwcm92ZQADAQAAAQEAAQIA","enc_key":"lR69cNXFuB3zyjDI12syH1XC4sfaTo63Ylms/I8yO39OTvuDmngPln8pMTJEwq9v","request_signature":"Ut6iy+f+l/FeMgcim+s3jkQO86nxgqhWweOfqFWwQGi3gercjBnzhoMmzrb0i0Z79iXUwE56GwM/2mgHDRz+CA==","certificate":{"session_vk":"w+xC1R2fZDJeQW4lERxYsd3XVAYy7wOx5sKsIWdwlWw=","creation_time":1737671115293,"ttl_min":9,"signature":"APDPqXOr+HqHMw48qNwDMPezgvwDzojcIaBztBdOYuTaomY2pWBVyXbaFyNN+huz5f44ZgMG1D6PnF/H/OGgkADgFHRr36qyRWIXLGJ/iArkqpw1lMzlikMeDlBjTOheIw=="}}' -X POST http://0.0.0.0:2024/v1/fetch_key
 ``` -->
 
-Key servers expose a set of metrics via a Prometheus server running on port 9184. They can be viewed in raw form by calling
-```shell
-curl http://0.0.0.0:9184
-```
+Key servers expose a set of metrics via a Prometheus server running on port 9184. They can be viewed in raw form by calling `curl http://0.0.0.0:9184`
 or used in a data visualization and analytics tool like Grafana.
 
+Similarly, an health check endpoint can be used by calling
+`curl http://0.0.0.0:2024/health`.
+
+### Permissioned mode
+In this mode only Seal policies of allowlisted packages can be queried.
+This is the recommended option for B2B use cases.
+
+First, a master seed should be generated.
+Use the `seal-cli` tool for that using `cargo run --bin seal-cli gen-seed`.
+It outputs the secret master seed.
+
+Create a configuration file with `Permissioned` mode and no clients following the instructions from the [example](crates/key-server/key-server-config.yaml).
+Run the server using `cargo run --bin key-server` with the environment variable 
+`MASTER_SEED` set to the master secret seed generated by the `seal-cli` tool,
```

Comment:
you mean set MASTER_KEY..? i dont see MASTER_SEED anywhere in code?

Reply:
```
It should abort after printing a list of unassigned derived public keys (search for logs with the text `Unassigned derived public key`).
```
i didn't understand this part on this first read, i also didnt get to abort - should this be a separate command?

Reply:
Will wait to suggest once Joy's comment is addressed

Reply:
>you mean set MASTER_KEY..? i dont see MASTER_SEED anywhere in code?

see the code of server.rs

> i didn't understand this part on this first read, i also didnt get to abort - should this be a separate command?

did you run the code from this branch?